SPECS/nexus-oss.spec

```diff
@@ -121,8 +121,12 @@ fi
 %preun
 %if 0%{?use_systemd}
 %if 0%{?suse_systemd}
+%if 0%{?suse_version} > 1500
+%service_del_postun_without_restart %{name}.service
```

Comment:
Wrong place, I will create a new PR (the CI did not complain as we don't test Tumbleweed ATM)